### PR TITLE
Front payment creditcard

### PR DESCRIPTION
**Pull request by sjpadgett**

### CODE REVIEW

**Comment by sjpadgett** [comment]
no idea why keep getting old merged commits!

**Comment by stephenwaite** [comment]
thanks for bringing this back to 5.0.2

**Comment by sjpadgett** [comment]
AuthorizeNet
image

image
image

Stripe: Only difference is card modal
image

More to come:
- Allow saving patient as gateway accounts for future transactions.
- Implement credit card readers for In Person transactions. (Verisign P-400).
- Currently portal patient payments are immediately charged. Will allow a hold (optional global) on authorized charge i.e pending until admin authorizes the charge in Dashboard audit. (up to 7 days)
- Stripe automatic email transaction receipt.
- Add Square gateway support. (If enough ask for it).
- After in person transaction work flow is done, will bring into v5.0.3

Main reason for bringing to front payments are for folks that don't have POS(Stripe doesn't provided) or will be handy for over phone/email or portal in house invoice payments(for those sending invoice from Fees->Posting to portal for patient to pay from patient documents(portal online payments  turned off globals)).
In other words, we're going to get our doggone money, somehow.:)

This is testing well enough to bring in.  Enjoy!
